frontend/cypress/tests/zoomView/zoomView_selectFileTypes.js

```diff
@@ -44,10 +44,67 @@ describe('check file types ', () => {
         .uncheck()
         .should('not.be.checked');
 
-    cy.get('.zoom__day > .commit-message > .fileTypeLabel')
-        .should('not.contain.text', 'java');
+    cy.get('#tab-zoom .fileTypes input[value="all"]')
+        .should('not.be.checked');
+  });
+
+  it('a commit of a specific file type should not be seen when that type is unchecked', () => {
```

Comment:
Maybe putting as `a commit of only one file type...` will be clearer for people who look at this for the first time?

Reply:
What do you think about `a commit should not be seen when all of its file types are unchecked`?

In this way the condition is not restricted to having only one file type.